tests/puppeteer/pages/BO/international/taxes/taxRules/index.js

```diff
@@ -0,0 +1,26 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class TaxRules extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Tax Rules • prestashop';
```

Comment:
```suggestion
    this.pageTitle = 'Tax Rules • ';
```
The `prestashop` part is the name of the shop :)